lib/toggle.cpp

```diff
@@ -22,6 +22,7 @@
 #include <config.h>
 #endif
 
+#include <iostream>
```

Comment:
OK, the code references the standard streams directly.  Will fix this together with the other one and attribute this to you.